### PR TITLE
bugfix/small-fixes-ole

### PR DESCRIPTION
**Pull request by olebowle**

### CODE REVIEW

File: modules/ProvBase/Entities/Modem.php [hunk 1]
```diff
@@ -264,11 +264,6 @@ public function tree()
         return $this->belongsTo(\Modules\HfcReq\Entities\NetElement::class);
     }
 
-    public function nelelement()
```
Comment (cschra) [inline modules/ProvBase/Entities/Modem.php]: > ```php   
> // TODO: deprecated! use netelement function instead - search for all places where this function is used
> public function tree()
> {
>     return $this->belongsTo(\Modules\HfcReq\Entities\NetElement::class);
> }
> ```
As by this comment it is advised to switch to the deleted function - either delete the comment or switch to the netelement function as recommended :wink:

Comment (NinoRy) [reply modules/ProvBase/Entities/Modem.php]: looks like tree function is also not used. Can be deleted as well.

File: app/Http/Controllers/Auth/LoginController.php [hunk 2]
```diff
@@ -101,7 +101,7 @@ protected function authenticated(Request $request, $user)
 
         self::setDashboardNotifications();
 
-        $user->update(['last_login_at' => Carbon::now()]);
+        App\User::where('id', $user->id)->update(['last_login_at' => Carbon::now()]);
```
Comment (olebowle) [inline app/Http/Controllers/Auth/LoginController.php]: @cschra Should we rather user ->save()? Did you want to circumvent the observer?

Comment (olebowle) [reply app/Http/Controllers/Auth/LoginController.php:104]: We decided to circumvent the observer.

**Pull request merged**